camera:
  color_topic: "/rover_camera/color"
  gray_topic: "/rover_camera/gray"
  info_topic: "/rover_camera/info"

c922:
  width: 640
  height: 480
  distortion_model: "plumb_bob"
<<<<<<< HEAD
  fx: 627.2
  fy: 630.6
  cx: 295
  cy: 237.1
=======
  K:
    fx: 627.2
    fy: 630.6
    cx: 295
    cy: 237.1
>>>>>>> d57da7d6ca9079783623660e902043048eed3e61
  D: [0.02, -0.114, 0.004, -0.017, 0.13]
  